docs/src/appendix/symmetric_tutorial.md

```diff
@@ -0,0 +1,1289 @@
+tensor product representation of $\mathbb{Z}_2$. This implies that, recalling [the
+introduction on symmetries](@ref ss_symmetries), in an appropriate basis
+for the local physical vector space, our local tensors would become block-diagonal where
+each so-called *matrix block* is labeled by a $\mathbb{Z}_2$ irrep.The appropriate local basis transformation is precisely the one that brings the local representation $X$ into block-diagonal form. Clearly, this transformation is nothing more than the Hadamard transformation which maps the computational basis of $Z$ eigenstates $\{\ket{\uparrow}, \ket{\downarrow}\}$ to that of the $X$
+eigenstates $\{\ket{+}, \ket{-}\}$ defined as $\ket{+} = \frac{\ket{\uparrow} +
+\ket{\downarrow}}{\sqrt{2}}$ and $\ket{-} = \frac{\ket{\uparrow} -
+\ket{\downarrow}}{\sqrt{2}}$. In the current context, this basis is referred to as the
+*irrep basis* of $\mathbb{Z}_2$, since each basis state corresponds to a one-dimensional
+irreducible representation of $\mathbb{Z}_2$. Indeed, the local symmetry transformation $X$
+acts trivially on the state $\ket{+}$, corresponding to the *trivial irrep*, and yields a
+minus sign when acting on $\ket{-}$, corresponding to the *sign irrep*.
+
+Next, let's make the statement that 'the matrix blocks of the local tensors are labeled by
+$\mathbb{Z}_2$ irreps' more concrete. To this end, consider the action of $ZZ$ in the irrep
+basis, which is given by the four nonzero matrix elements
+
+```math
+\begin{align}
+\label{eq:zz_matel}
+ZZ : \mathbb C^2 \otimes \mathbb C^2 &\to \mathbb C^2 \otimes \mathbb C^2 : \\
+\ket{+} \otimes \ket{+} &\mapsto \ket{-} \otimes \ket{-}, \nonumber \\
+\ket{+} \otimes \ket{-} &\mapsto \ket{-} \otimes \ket{+}, \nonumber \\
+\ket{-} \otimes \ket{+} &\mapsto \ket{+} \otimes \ket{-}, \nonumber \\
+\ket{-} \otimes \ket{-} &\mapsto \ket{+} \otimes \ket{+}. \nonumber \\
+\end{align}
+```
+
+We will denote the trivial $\mathbb{Z}_2$ irrep by $'0'$, corresponding to a local $\ket{+}$
+state, and the sign irrep by $'1'$, corresponding to a local $\ket{-}$ state. Given this
+identification, we can naturally associate the tensor product of basis vectors in the irrep
+basis to the tensor product of the corresponding $\mathbb{Z}_2$ irreps. One of the key
+questions of the [representation theory of groups](representation_theory) is how the tensor
+product of two irreps can be decomposed into a direct sum of irreps. This decomposition is
+encoded in what are often called the
+[*fusion rules*](https://en.wikipedia.org/wiki/Fusion_rules),
+```math
+a \otimes b \cong \bigoplus_c N_c^{ab} c,
+```
+where $N_{ab}^c$ encodes the number of times the irrep $c$ occurs in the tensor product of
+irreps $a$ and $b$. These fusion rules are called *Abelian* if the tensor product of any two
+irreps corresponds to exactly one irrep. We will return to the implications of *non-Abelian*
+fusion rules [later](@ref ss_non_abelian). For the case of the $\mathbb{Z}_2$ irreps, the fusion
```

Comment:
In the `FusionStyle` trait, we are not actually using the "abelian" vs "non-abelian" nomenclature, because at some point it became confusing if you also want to consider Vec[G]. For a non-abelian group, the "fusion" of the simple objects of Vec[G] is unique. However, it does depend on the order: `a ⊗ b != b ⊗ a`. Hence, there cannot be a braiding. On the other hand, for `Rep[G]`, it does hold that `a ⊗ b == b ⊗ a`, also for a non-abelian group.

So then, what does abelian mean: does it mean unique fusion, or independent of the order fusion? 

Note that I have no problem with calling the group abelian or non-abelian; but I would prefer to avoid calling fusion rules abelian.